core/src/components/button/test/theme-ionic/button.e2e.ts

```diff
@@ -0,0 +1,82 @@
+import { expect } from '@playwright/test';
+import type { E2EPage } from '@utils/test/playwright';
+import { configs, test } from '@utils/test/playwright';
+
+class ButtonFixture {
+  readonly page: E2EPage;
+  readonly screenshotFn?: (file: string) => string;
+
+  constructor(page: E2EPage, screenshot?: (file: string) => string) {
+    this.page = page;
+    this.screenshotFn = screenshot;
+  }
+
+  async selectOption(select: string, optIndex: number) {
+    const selectElem = this.page.locator(`ion-select#select-${select}`);
+    await selectElem.click();
+
+    const selectOptElem = this.page.locator(`ion-alert .alert-radio-group button:nth-child(${optIndex + 1})`);
+    await selectOptElem.click();
+
+    const selectOkBtn = this.page.locator(`ion-alert .alert-button-group button:nth-child(2)`);
+    await selectOkBtn.click();
+  }
+
+  async checkScreenshot(modifier: string) {
+    const { screenshotFn } = this;
+
+    if (!screenshotFn) {
+      throw new Error(
+        'A screenshot function is required to take a screenshot. Pass one in when creating ButtonFixture.'
+      );
+    }
+
+    await expect(this.page).toHaveScreenshot(screenshotFn(`${modifier}-button`));
+  }
+}
+
+configs({ themes: ['ionic'] }).forEach(({ config, screenshot, title }) => {
```

Comment:
```suggestion
configs({ themes: ['ionic'], modes: ['md'] }).forEach(({ config, screenshot, title }) => {
```

Since the appearance and behavior we are testing for the button is consistent between platforms, we can limit the testing to a specific mode.

Reply:
Done!